src/utilities/getSquaredNumbers/getSquaredNumbers.test.js

```diff
@@ -0,0 +1,120 @@
+import { getSquaredNumbers } from './getSquaredNumbers';
+
+describe('getSquaredNumbers()', () => {
+  describe('when provided with valid arrays of numbers', () => {
+    describe('and array contains positive integers', () => {
+      it('should return [1, 4, 9, 16, 25] for [1, 2, 3, 4, 5]', () => {
+        const result = getSquaredNumbers([1, 2, 3, 4, 5]);
+        expect(result).toEqual([1, 4, 9, 16, 25]);
+      });
+    });
+
+    describe('and array contains negative numbers', () => {
+      it('should handle mixed positive and negative numbers', () => {
```

Comment:
Here you're repeating yourself quite a bit.

and array contains negative numbers
should handle mixed positive and negative numbers